src/index.html

```diff
@@ -1,13 +1,309 @@
+            </div>
+          </div>
+
+          <div class="details__detail detail">
+            <div class="detail__photos">
+              <div class="grid grid--details">
+                <div class="grid__item--details--1-2">
+                  <div class="detail__container">
+                    <img
+                      src="images/photos/photo3.png"
+                      alt="photo3"
+                      class="detail__photo"
+                    >
+                  </div>
+                </div>
+
+                <div class="grid__item--details--3-3">
+                  <div class="detail__container">
+                    <img
+                      src="images/photos/photo4.png"
+                      alt="photo4"
+                      class="detail__photo"
+                    >
+                  </div>
+                </div>
+              </div>
+            </div>
+
+            <div class="detail__description">
+              <div class="detail__subtitle">Range & Integrated lights</div>
+              The removable battery has up to 70km battery autonomy and weighs only 2.4 kg. Lights integrated into the frame give you always-on visibility day and night.
+            </div>
+          </div>
+
+          <div class="details__detail detail">
+            <div class="detail__photos">
+              <div class="grid grid--details">
+                <div class="grid__item--details--1-1">
+                  <div class="detail__container">
+                    <img
+                      src="images/photos/photo5.png"
+                      alt="photo5"
+                      class="detail__photo"
+                    >
+                  </div>
+                </div>
+
+                <div class="grid__item--details--2-3">
+                  <div class="detail__container">
+                    <img
+                      src="images/photos/photo6.png"
+                      alt="photo6"
+                      class="detail__photo"
+                    >
+                  </div>
+                </div>
+              </div>
+            </div>
+
+            <div class="detail__description">
+              <div class="detail__subtitle">Hydraulic disc brakes & Lightweight</div>
+              Brakes with total stopping power the second you make contact. The removable battery has up to 70km battery autonomy and weighs only 2.4 kg. Lights integrated into the frame give you always-on visibility day and night.
+            </div>
+          </div>
+
+
+          <div class="grid">
+            <a href="#" class="button">Explore</a>
+          </div>
+        </div>
+      </section>
+
+      <section class="page__section contact-us" id="contacts">
+        <div class="container">
+          <h2 class="page__section-title">Contact us</h2>
+          <div class="grid grid--contact-us">
+            <div class="grid__item--contact-us--1-5">
+              <form action="#" method="post">
```

Comment:
```suggestion
              <form action="#" method="GET">
```
Поміняй метод на get. Зараз отримуємо помилку при сабміті